srcopsmetrics/entities/__init__.py

```diff
@@ -16,6 +16,6 @@
 # along with SrcOpsMetrics.  If not, see <http://www.gnu.org/licenses/>.
 
 """Entities imports."""
+from .interface import Entity
```

Comment:
import needed?

Reply:
please solve pre-commit :)

Reply:
absolutelly forgot about this one
import is needed as everything else besides `entities` module uses only Entity import alias `from entities import Entity`
i've just added it to __all__ and seems like pre-commit is fine w/ it